### DISCUSSION TITLE
iNAV 5.1.0 OSD issues

### DISCUSSION THREAD

**Discussion by JimValo** [iNavFlight/inav#8460] [Q&A]
I have a Matek F411-WTE FC and in iNAV I set my warning altitude at 400 feet when I fly in my goggles the altitude is there until I reach 400 feet then it completely disappears and I see nothing on the altitude but then when I go below 400 feet the altitude comes back why I think it’s supposed to just flash but it disappears?

**Comment by OptimusTi** [comment]
There are some boards that do not support the "blink" feature of the MAX OSD chip. You can fix this by setting the following command in the CLI: set display_force_sw_blink = ON

**Reply by JimValo** [reply]
I will check it out and let you know. 
thanks

**Comment by JimValo** [comment]
I did check FC and the Cli: display_force_sw_blink was turned off I did turn it on but the weather has stopped me from testing it so I will let you know after I test it out. 
thanks again
